Sentry.xcodeproj/project.pbxproj

```diff
@@ -2005,7 +2005,6 @@
 				844DA80728246D5000E6B62E /* Gemfile */,
 				844DA80428246D5000E6B62E /* Makefile */,
 				844DA81F28246DE300E6B62E /* scripts */,
-				844DA80928246D5000E6B62E /* dangerfile.js */,
```

Comment:
Why'd this get removed?

Reply:
This file don't exist. I just removed the reference

Reply:
Oh, LOL 😆